src/engine/strat_engine/backstore/crypt.rs

```diff
@@ -599,6 +583,31 @@ impl CryptHandle {
     }
 }
 
+/// Interpret non-Clevis keys that may contain additional information about
+/// how to configure Clevis when binding. Remove any expected non-Clevis keys
+/// from the configuration and return it.
+/// The only value to be returned is whether or not the bind command should be
+/// passed the argument yes, which is returned in the first item of the
+/// returned pair.
+pub fn interpret_clevis_config(pin: &str, mut clevis_config: Value) -> Result<(bool, Value)> {
```

Comment:
I think a prototype of `&str, &mut Value` would make more sense here, remove the key-value pairs through the mutable reference, and then only return the data that has been parsed out of the `Value`.

Reply:
> I think a prototype of `&str, &mut Value` would make more sense here, remove the key-value pairs through the mutable reference, and then only return the data that has been parsed out of the `Value`.

Done!